ASoC: Intel: soc-acpi-intel-ptl-match: Add rt712_vb and rt1320 support

macchian opened the pull request:
This patch adds support for the rt712_vb multi-function codec and rt1320 amplifier. 
oth devices are connected via a single Soundwire link 3, sharing the sdw3-clock, sdw3-data[0] and sdw3-data[1] data lanes
respectively.

plbossart commented on sound/soc/intel/common/soc-acpi-intel-ptl-match.c:
```diff
@@ -69,6 +101,24 @@ static const struct snd_soc_acpi_adr_device rt711_sdca_0_adr[] = {
+		.adr = 0x000330025D132001ull,
+		.num_endpoints = 1,
+		.endpoints = &spk_r_endpoint,
+		.name_prefix = "rt1320-1"
```
is there a need for the '-1' suffix?

macchian replied:
> is there a need for the '-1' suffix?
yes, we checked here. https://github.com/thesofproject/linux/blob/c2fd38c76c0ac19946a62349fd53a45948a43181/sound/soc/sdw_utils/soc_sdw_rt_amp.c#L205

macchian replied:
> is there a need for the '-1' suffix?
yes, we checked the substring  at https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sdw_utils/soc_sdw_rt_amp.c#L205

plbossart commented:
superseded by #5315 @macchian @ujfalusi ?

macchian commented:
> superseded by #5315 @macchian @ujfalusi ?
RT713 is jack codec with individual hardware revision id. Suppose to be a separated configuration from rt712.

ranj063 commented:
@macchian is this PR still relevant?

macchian commented:
yes, I will rebase the PR and submit for reviewing. The 712_l3_1320_l3 are both probed successfully.

macchian committed:
ASoC: Intel: soc-acpi-intel-ptl-match: Add rt713_vb_l3_rt1320_l3 support

This patch adds support for the rt712_vb multi-function codec and
rt1320 amplifier. Both devices are connected via a single Soundwire link 3,
sharing the sdw3-clock, sdw3-data[0] and sdw3-data[1] data lanes
respectively.

Signed-off-by: Mac Chiang <mac.chiang@intel.com>

macchian committed:
ASoC: Intel: soc-acpi-intel-lnl/ptl-match: Simplify variable name

The variable name contains duplications;thus, remove the redundant strings.

Signed-off-by: Mac Chiang <mac.chiang@intel.com>
